src/admin-views/settings/emails/message-templates/list-icon.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Tickets Emails Message Template List Icon
+ *
+ * @since  TBD   Icon (checkmark) for list item of email message templates for Emails settings tab.
+ * 
+ * @var Array[]  $templates  Array of template info.
+ * @var Array    $template   Template info.
+ */
+
+// @todo $templates variable will be an array of Message_Template objects in the future.
+// @todo $template variable will be a Message_Template object in the future.
+
+// If no template, bail.
+if ( empty( $template ) ) {
+	return;
+}
+
+$icon_classes   = [ 'dashicons' ];
+$icon_classes[] = tribe_is_truthy( $template['enabled'] ) ? 'dashicons-yes' : '';
+
+?>
+<div class="tec-tickets__admin-settings-emails-template-list-item-icon">
+	<span <?php tribe_classes( $icon_classes ); ?> ></span>
```

Comment:
should we use an `<i>` tag here?

Reply:
I guess we could. I used `<span>` since `<i>` is technically for `italic`, not really `icon`. On the WordPress Dashicons page, they use `<span>` in their examples. Semantically, I think `<span>` is the way to go, but if we have a policy that says to use `<i>` instead, I can always change it.

Reply:
perfecto!